_data/menudata.yml

```diff
@@ -1,4 +1,4 @@
 about: ['organization', 'partners', 'terms_of_reference', 'acknowledgement']
 nodes: ['cadc', 'euclid', 'inaf-oat', 'jcmt_gbs', 'jwst', 'ngvs', 'nugrid', 'uvic', 'vlass']
-docs: ['quick_start', 'storage', 'group_management', 'digital_object_identifiers', 'openstack_cloud_portal', 'cloud_migration', 'caom_query_service', 'jcmt_starlink']
+docs: ['quick_start', 'storage', 'group_management', 'digital_object_identifiers', 'openstack_cloud_portal', 'caom_query_service', 'jcmt_starlink']
```

Comment:
Should the first entry in 'doc' read 'overview' instead of 'quick_start', given that the latter has been removed from translations.yml? (or does that related to the filename quick_start_en.md?) 

Similar question for caom_query_service v cloud_migration.